app/frontend/src/pages/QualifierPage/QualifierPage.tsx

```diff
@@ -9,35 +9,39 @@ import {
   logoStacked,
   logoStackedOnDark,
 } from "assets/images/images";
-import { QualifierPageCalendar } from "./QualifierPageCalendar";
+const QualifierPageCalendar = React.lazy(
+  () => import("./QualifierPageCalendar"),
+);
 
-function QualifierPage() {
+function QualifierPage(): JSX.Element {
```

Comment:
Not sure this casting is needed, since the return for the function is rather explicit already, so for most scenarios both machine and human would not interpret this function to returning anything besides an element. Thoughts?

Reply:
Yes, you're correct. Making the edits now.